packages/apps/patterns/react-appkit/src/components/Fallback/Fallback.tsx

```diff
@@ -19,3 +21,35 @@ export const GenericFallback = () => {
   const { t } = useTranslation('appkit');
   return <Fallback message={t('generic loading label', { ns: 'appkit' })} />;
 };
+
+export const ClientFallback = ({ client, status }: Partial<ClientContextProps>) => {
```

Comment:
One line comment.
I assume we can't use the hook since this is used before the context is set-up. In which case pass in needed properties (compatReconnect) rather than the client object.